P3826/P3826.md

```diff
@@ -169,7 +169,7 @@ customization hooks are yet?
 
 To answer that question for myself, I implemented new customization hooks
 [here](https://github.com/NVIDIA/cccl/pull/5793) that address the known issues. Using that
-design (described [here](#the-planned-fix)) as a polestar, this paper proposes wording
+design (described in [](#the-planned-fix)) as a polestar, this paper proposes wording
```

Comment:
This is the feature that I've implemented in mpark framework. If you don't use any text in the link and you want to have a cross-reference to some id in the paper mpark-wg21 will find this id for you and extract the text and put it as a cross-reference text.

It should also report an error (or warning) if this id is not found.

Let me know if you like it, or if you prefer the old fashion way. You should update mpark (submodule in your case, I guess) if you have not done it recently

Reply:
i think i need to point my mpark-wg21 submodule to a more recent commit. this syntax doesn't work for me yet